doc/ExternalProvisioning.md

```diff
@@ -0,0 +1,64 @@
+# External provisioning in IoT Edge
+
+IoT Edge supports a provisioning mode called `external` whereby the `iotedged` obtains the device's provisioning information by calling a REST API hosted on an HTTP endpoint.
+The endpoint that is targeted by the `iotedged` is specified in IoT Edge's config.yaml file as follows:
+
+```yaml
+provisioning:
+  source: "external"
+  endpoint: "http://localhost:9999"
+```
+
+**Please note that HTTPS is currently not supported for this external endpoint.**
+
+**Customers who would like to restrict access to the external endpoint to just whitelist `iotedged` can do so by hosting their HTTP endpoint on Unix sockets instead of using TCP and giving appropriate permissions to the `iotedged` to be able to access the socket.**
```

Comment:
Just to be clear, this is saying to use `unix://<>` for `provisioning.endpoint`, right? If so, add a full example of that just like the `http` example.

Reply:
Done